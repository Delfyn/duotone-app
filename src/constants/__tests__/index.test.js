import * as actionTypes from '../index';

test('actionTypes are exported correctly', () => {
  expect(actionTypes).toMatchSnapshot();
});
