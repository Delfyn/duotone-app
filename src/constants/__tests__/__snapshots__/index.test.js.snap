// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actionTypes are exported correctly 1`] = `
Object {
  "actionTypes": Object {
    "ADD_DUOTONE_COLOR": "ADD_DUOTONE_COLOR",
    "ADD_FILE": "ADD_FILE",
    "INVALID_FILE": "INVALID_FILE",
    "SET_DUOTONE_COLOR": "SET_DUOTONE_COLOR",
    "SET_PRIMARY_COLOR": "SET_PRIMARY_COLOR",
    "SET_SECONDARY_COLOR": "SET_SECONDARY_COLOR",
    "TOGGLE_COLOR_PICKER": "TOGGLE_COLOR_PICKER",
  },
}
`;
